Algorithm.Framework/Portfolio/ConfidenceWeightedPortfolioConstructionModel.cs

```diff
@@ -29,6 +30,24 @@ namespace QuantConnect.Algorithm.Framework.Portfolio
     /// </summary>
     public class ConfidenceWeightedPortfolioConstructionModel : InsightWeightingPortfolioConstructionModel
     {
+        /// <summary>
+        /// Initialize a new instance of <see cref="InsightWeightingPortfolioConstructionModel"/>
+        /// </summary>
+        /// <param name="rebalancingFunc">Rebalancing function</param>
```

Comment:
I think we need a bit more documentation on this overload, specifically what the function is expected to do, eg: 'for a given algorithm UTC DateTime return the next expected rebalance time'